Userland/Games/Chess/Engine.cpp

```diff
@@ -54,13 +54,11 @@ void Engine::connect_to_engine_service()
     close(wpipefds[0]);
     close(rpipefds[1]);
 
-    auto infile = Core::DeprecatedFile::construct();
-    infile->open(rpipefds[0], Core::OpenMode::ReadOnly, Core::DeprecatedFile::ShouldCloseFileDescriptor::Yes);
-    set_in(infile);
+    auto infile = Core::File::adopt_fd(rpipefds[0], Core::File::OpenMode::Read, Core::File::ShouldCloseFileDescriptor::Yes).release_value_but_fixme_should_propagate_errors();
```

Comment:
As discussed on Discord, the `ShouldCloseFileDescriptor` argument can be left to its default if you'd like.